electron/src/menu/TrayHandler.ts

```diff
@@ -98,8 +98,19 @@ export class TrayHandler {
 
   private flashApplicationWindow(win: BrowserWindow, count?: number): void {
     if (win.isFocused() || !count) {
-      win.flashFrame(false);
-    } else if (count > this.lastUnreadCount) {
+      if (process.platform === 'win32') {
+        win.flashFrame(false);
+      }
+      return;
+    }
+    /* After an Electron API change https://github.com/electron/electron/pull/41391
+       flashFrame() leads to a constant bouncing of the dock icon on macOS.
+       By calling the dock.bounce() directly, we avoid this behavior. the "informational"
+       is optional (default), but makes it easier to read
+    */
+    if (process.platform === 'darwin') {
+      app.dock.bounce('informational');
+    } else if (process.platform === 'win32') {
```

Comment:
woops we missed this

```suggestion
    } else if (count > this.lastUnreadCount) {
```